extensions/quarkus/deployment/src/main/java/org/apache/myfaces/core/extensions/quarkus/deployment/MyFacesProcessor.java

```diff
@@ -788,10 +792,20 @@ else if (ai.target().asMethod().returnType().kind() == Type.Kind.PARAMETERIZED_T
             // register type
             reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, typeName));
             // and try to register the ClientProxy
-            reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, typeName + "_ClientProxy"));
+            if (shouldProxy(typeName))
+            {
+                reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, typeName + "_ClientProxy"));
+            }
         }
     }
 
+    boolean shouldProxy(String typeName)
+    {
+        return !(typeName.startsWith("java.lang") 
+                 || typeName.startsWith("java.util") 
+                 || typeName.startsWith("jakarta.faces"));
```

Comment:
Why skip jakarta.faces? We produce named beans

Reply:
It's only for the ClientProxy versions of the beans and I dumped out from Quarkus which jakarta beans were found and which were alerted to not have ClientProxy objects.   I will run some more tests.

Reply:
In theory each bean with scope != Dependent should have a proxy